app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/ShowTicketActivity.java

```diff
@@ -33,13 +33,22 @@ public void onCreate(Bundle savedInstanceState) {
         moviedetailTextView = (TextView)findViewById(R.id.moviedetail);
         ticketsnumberTextView = (TextView)findViewById(R.id.ticketnumber);
         ticketqrcode = (ImageView) findViewById(R.id.ticket_qrcode);
-        //get data from last activity
+        //Get data from KinoDetailsFragment
         Intent intent = getIntent();
         String data = getIntent().getStringExtra("movie_data");
         moviedetailTextView.setText(data);
-        ticketsnumberTextView.setText("87237489273984");//data from backend
 
-        String text="KingsMan 08.05 Kailiang Dong "; // Whatever you need to encode in the QR code
+        //TODO:It is data from backend. Wait for setting up of backend
+        ticketsnumberTextView.setText("87237489273984");
+
+        // TODO:it is Whatever you need to encode in the QR code.Wait for setting up of backend
+        String text="KingsMan 08.05 Kailiang Dong ";
```

Comment:
The ticket code that will be transformed to the QR code is a random 128 character string. Name and stuff like this will not be part of it.

Reply:
Fixed. I deleted mock-up text. Add annotation here
        String text = "";//Here should be a random 128 character string